easybuild/easyconfigs/r/R/R-3.6.0-foss-2019a.eb

```diff
@@ -0,0 +1,2123 @@
+name = 'R'
+version = '3.6.0'
+biocver = '3.9'
```

Comment:
@fizwit `biocver` isn't used anywhere, can be removed?

Reply:
I’ll remove it, but for locally we use it for adding BioConductor packages. Would be nice to support biocver in the future.

Reply:
removed,